src/Tizen.NUI/src/public/Theme/ThemeManager.cs

```diff
@@ -52,7 +53,7 @@ public static class ThemeManager
         private static Theme defaultTheme;
         private static readonly List<Theme> builtinThemes = new List<Theme>(); // Themes provided by framework.
         private static readonly List<Theme> customThemes = new List<Theme>(); // Themes registered by user. (Legacy support)
-        private static readonly List<string> packages = new List<string>();
+        private static readonly List<IThemeCreator> packages = new List<IThemeCreator>();
```

Comment:
if possible, could you remove the warning below?

src/Tizen.NUI/src/public/Theme/ThemeManager.cs#L58
Initialize all static fields in 'ThemeManager' when those fields are declared and remove the explicit static constructor

Reply:
warning message `Initialize all static fields in 'ThemeManager' when those fields are declared and remove the explicit static constructor` is not related this commit. 
To keep atomic commits, i think we can handle this warning in another patch.